pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -770,12 +746,16 @@ func (s *DiscoveryServer) removeEdsCon(clusterName string, node string, connecti
 	}
 }
 
-// FetchEndpoints implements xdsapi.EndpointDiscoveryServiceServer.FetchEndpoints().
-func (s *DiscoveryServer) FetchEndpoints(ctx context.Context, req *xdsapi.DiscoveryRequest) (*xdsapi.DiscoveryResponse, error) {
-	return nil, errors.New("not implemented")
-}
+func endpointDiscoveryResponse(loadAssignments []*xdsapi.ClusterLoadAssignment) *xdsapi.DiscoveryResponse {
+	out := &xdsapi.DiscoveryResponse{
+		TypeUrl:     EndpointType,
```

Comment:
Can you also keep the comments from `endpoints`?

Reply:
> // All resources for EDS ought to be of the type ClusterLoadAssignment	

This is useless

Reply:
The other one

Reply:
done